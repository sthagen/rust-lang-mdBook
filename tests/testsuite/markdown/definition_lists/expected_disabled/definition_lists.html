<h1 id="definition-lists"><a class="header" href="#definition-lists">Definition Lists</a></h1>
<p>apple
:   red fruit</p>
<p>orange
:   orange fruit</p>
<p>apple
:   red fruit
:   computer company</p>
<p>orange
:   orange fruit
:   telecom company</p>
<p>term
:   1. Para one</p>
<pre><code>   Para two
</code></pre>
<h2 id="term-with-link"><a class="header" href="#term-with-link">Term with link</a></h2>
<p><a href="some-page.html#apple">apple</a>
: red fruit</p>
<h2 id="multi-line-term"><a class="header" href="#multi-line-term">Multi-line term</a></h2>
<p>a
b<br>c</p>
<p>:   foo</p>
<h2 id="nested"><a class="header" href="#nested">Nested</a></h2>
<p>level one
: l1
level two
: l2
level three
: l3</p>
<p>level one
: l1</p>
<h2 id="loose"><a class="header" href="#loose">Loose</a></h2>
<p>apple</p>
<p>:   red fruit
:   computer company</p>
<p>orange</p>
<p>:   orange fruit</p>