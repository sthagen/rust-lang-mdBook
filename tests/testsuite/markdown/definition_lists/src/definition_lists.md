# Definition Lists

apple
:   red fruit

orange
:   orange fruit

apple
:   red fruit
:   computer company

orange
:   orange fruit
:   telecom company

term
:   1. Para one

       Para two

## Term with link

[apple](some-page.md#apple)
  : red fruit

## Multi-line term

a
b\
c

:   foo

## Nested

level one
: l1
    level two
    : l2
        level three
        : l3

level one
: l1

## Loose

apple

:   red fruit
:   computer company

orange

:   orange fruit
