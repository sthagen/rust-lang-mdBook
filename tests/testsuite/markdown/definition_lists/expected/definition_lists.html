<h1 id="definition-lists"><a class="header" href="#definition-lists">Definition Lists</a></h1>
<dl>
<dt id="apple"><a class="header" href="#apple">apple</a></dt>
<dd>red fruit</dd>
<dt id="orange"><a class="header" href="#orange">orange</a></dt>
<dd>orange fruit</dd>
<dt id="apple-1"><a class="header" href="#apple-1">apple</a></dt>
<dd>red fruit</dd>
<dd>computer company</dd>
<dt id="orange-1"><a class="header" href="#orange-1">orange</a></dt>
<dd>orange fruit</dd>
<dd>telecom company</dd>
<dt id="term"><a class="header" href="#term">term</a></dt>
<dd>
<ol>
<li>
<p>Para one</p>
<p>Para two</p>
</li>
</ol>
</dd>
</dl>
<h2 id="term-with-link"><a class="header" href="#term-with-link">Term with link</a></h2>
<dl>
<dt id="apple-2"><a class="header" href="#apple-2"><a href="some-page.html#apple">apple</a></a></dt>
<dd>red fruit</dd>
</dl>
<h2 id="multi-line-term"><a class="header" href="#multi-line-term">Multi-line term</a></h2>
<dl>
<dt id="a-bc"><a class="header" href="#a-bc">a
b<br>c</a></dt>
<dd>
<p>foo</p>
</dd>
</dl>
<h2 id="nested"><a class="header" href="#nested">Nested</a></h2>
<dl>
<dt id="level-one"><a class="header" href="#level-one">level one</a></dt>
<dd>
<dl>
<dt id="l1-level-two"><a class="header" href="#l1-level-two">l1
level two</a></dt>
<dd>
<dl>
<dt id="l2-level-three"><a class="header" href="#l2-level-three">l2
level three</a></dt>
<dd>l3</dd>
</dl>
</dd>
</dl>
</dd>
<dt id="level-one-1"><a class="header" href="#level-one-1">level one</a></dt>
<dd>l1</dd>
</dl>
<h2 id="loose"><a class="header" href="#loose">Loose</a></h2>
<dl>
<dt id="apple-3"><a class="header" href="#apple-3">apple</a></dt>
<dd>
<p>red fruit</p>
</dd>
<dd>
<p>computer company</p>
</dd>
<dt id="orange-2"><a class="header" href="#orange-2">orange</a></dt>
<dd>
<p>orange fruit</p>
</dd>
</dl>